test.config.js

```diff
@@ -44,4 +44,4 @@ sd.AP = {
   twitterPath: "/twitter",
 }
 sd.APP_URL = "http://artsy.net"
-sd.RECAPTCHA_KEY = "RECAPTCHA_KEY"
+sd.RECAPTCHA_KEY = "6LdvSaIUAAAAAD92qOsXbAEaBpBfLmWfd9_T-J59"
```

Comment:
Repeated?

Reply:
Went back to the original stub here and removed from the test env.